main/src/library/Concurrent/Channel.flix

```diff
@@ -171,30 +169,71 @@ namespace Concurrent/Channel {
         unlock(channelLock);
         element
 
+    ///
+    /// Receives an element from the channel `c`. This will crasah if the
+    /// channel is empty.
+    ///
+    @Internal
+    pub def assertiveGet(c: Mpmc[a]): a \ IO = {
+        let Mpmc(admin, elementDeque) = c;
+        let MpmcAdmin(_, channelLock, _, _, size, _, _) = admin;
+        lock(channelLock);
+
+        // Try to get the element (which could already be taken by someone
```

Comment:
Lets phrase this as: Gets an element from the channel `c` which must be non-empty.